### DISCUSSION TITLE
I would like to use Filecoin network as storage for my next app. Can I do this?

### DISCUSSION THREAD

**Discussion by giacomo-giacomo** [filecoin-project/community#535] [Network + Implementations Announcements]
Hello FIL team. I have some questions for you. I build apps using Python. I generally store data in SQLite Databases. I would like to try Filecoin as storage provider so I am here asking these questions:
-Can Filecoin be used to store SQLIte files? 
-I should update or change data on my SQLite file, so I can update my database. Would it be technically possible with Filecoin?
-How much time is required to update an SQLite database stored on Filecoin Network?

Thank you for your answers.

**Comment by web3jenks** [comment]
Hi @astrid-net , Filecoin is best used for cold and archival data for long term persistence right now. We do have IPFS and storage helpers such as Web3.storage, NFT.storage and Estuary.tech for fast retrieval and could be used for hosting file based SQLlite database.

You can store and retrieve, but I don't know if there are enough tests done on this to show that IPFS is able to give a satisfactory performance. It also depends on where the database file etc, in a distributed system it performance might vary. 

We do have following tools that I encourage you to look into: 
- [Tableland](https://tableland.xyz/)
- [Textile](https://docs.textile.io/threads/)
- [Orbit DB](https://github.com/orbitdb/orbit-db)

I found some discussion on our public Filecoin slack that is related to what you are looking to do: 
https://filecoinproject.slack.com/archives/C016APFREQK/p1638190164495000
